### PR TITLE
Humberger menu

### PR DESCRIPTION
**Pull request by AbdimulhinYussuf3675**
My partner and I implemented the hamburger menu at this branch on desktop and mobile versions.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .eslintrc.json:1]: - [ ] Kindly setup linters for javascript. I can see your linters are checking your html and css files but kindly add your re-configure your linters to check javascript files using this [code of linters.yml](https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.github/workflows/linters.yml) file 😊

File: css/style.css [hunk 2]
```diff
@@ -376,8 +376,6 @@ input:focus:invalid {
   box-shadow: none;
 }
```
Comment (HAMMAS-SALEEM) [inline css/style.css:379]: ### Your Mobile Menu
menu-student

### Figma Mobile Menu
menu-figma

- [ ] Your mobile menu should have the same background-color as you can see in the Figma template to make your design look consistent with your template 👍 
- [ ] Your menu should blur everything behind your menu as you can see in the Figma template to make your design more professional and human friendly 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @AbdimulhinYussuf3675, @yash244466666  :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Following Github flow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json (hunk 1 above)
Comment (AbdimulhinYussuf3675) [reply .eslintrc.json:1]: @HAMMAS-SALEEM I have configured the linters but can't see it checking on GitHub. What do you think may have caused the problem?

Comment (AbdimulhinYussuf3675) [reply .eslintrc.json:1]: Solved, thanks

**Review by ichala — APPROVED** [review]
Hi @AbdimulhinYussuf3675 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemenetd ✔️  

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**